functions-methods.js

```diff
@@ -33,4 +70,23 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email){
+    if((email.includes("@")===true)
+        &&
+        (!(email[(email.length-1)]==='.'))
+        &&
+        (email.includes(",")===false)){
+        return true;
+    } else{
+        return false;
+    }
+}
+
+console.log(checkEmailValidity("n.eeken@novi.nl") );
+console.log(checkEmailValidity("tessmellink@novi.nl"));
+console.log(checkEmailValidity("n.eekenanovi.nl") );
+console.log(checkEmailValidity("n.eeken@novinl."));
+console.log(checkEmailValidity("tessmellink@novi,nl") )
+
```

Comment:
Voor de rest zijn de uitkomsten oké. Heb het met mijn code vergeleken en iedereen doet het anders maar de hoofdzaak is dat je je eigen code begrijpt toch. Vindt het ook moeilijk om andermans code helemaal uit te pluizen, daar gaat ook heel veel tijd in zitten. Bedankt nog dat je mijn huiswerk nog had nagekeken over de intermediate functies.